README.md

```diff
@@ -14,23 +14,61 @@ Clone this repo: <a href = "https://github.com/Swarathmica-infraspec/payout-mana
 GO-VERSION: 1.25.0
 
 The project contains payoutmanagementsystem/ <br>
-    - .github/workflows/payoutManagementSystem.yml <br>
-    - payee/ <br>
-        - payee.go <br>
-        - payee_test.go <br>
-    - go.mod <br>
-    - go.sum <br>
-    - README.md <br>
+
+- .github/workflows/payoutManagementSystem.yml <br>
+- payee/
+  - payee.go <br>
+  - payee_test.go <br>
+  - payee_db.sql <br>
+  - payeeDAO.go <br>
+  - payeeDAO_test.go <br>
+- go.mod <br>
+- go.sum <br>
+- README.md <br>
+
 
 NOTE: Only email ids with .com are supported.
 
-# Run Tests
 
-Test can be run by executing the below command in the terminal
-  go test -v ./...
+
+# Database Setup
+
+We use PostgreSQL running inside Docker for persistant storage.
+
+## 1. Start Postgres with Docker Compose
+
+From the project root, open VS Code. Press F1: Dev Containers: Reopen in Dev Container
```

Comment:
[question|possible-documentation-fix] do I need to install a plugin for dev containers in vscode or something?
the documentation needs update,
Alternatively is it possible to do devcontainer setup through command line, without the use of vscode for local development ?
@Swarathmica-infraspec  @KirthiInfra